src/test/java/org/jenkinsci/plugins/docker/commons/credentials/DockerServerCredentialsTest.java

```diff
@@ -57,7 +57,8 @@ public void configRoundTripEmpty() throws Exception {
                 Collections.<DomainSpecification>singletonList(new DockerServerDomainSpecification()));
         DockerServerCredentials credentials = new DockerServerCredentials(CredentialsScope.GLOBAL, "foo", "desc", "", "", "");
         store.addDomain(domain, credentials);
-        j.submit(j.createWebClient().goTo("credential-store/domain/" + domain.getName() + "/credential/"+credentials.getId()+"/update")
+
+        j.submit(j.createWebClient().goTo("credentials/store/system/domain/" + domain.getName() + "/credential/"+credentials.getId()+"/update")
```

Comment:
Yes, it just ends up throwing a 404.  I even checked on running copy of Jenkins too in order to see if there was a redirect that wasn't followed.  Nothing.